awx/api/serializers.py

```diff
@@ -98,21 +98,12 @@
                                            'total_hosts',
                                            'hosts_with_active_failures',
                                            'total_groups',
-                                           'groups_with_active_failures',
                                            'has_inventory_sources',
                                            'total_inventory_sources',
                                            'inventory_sources_with_failures',
                                            'organization_id',
                                            'kind',
                                            'insights_credential_id',),
-    'host': DEFAULT_SUMMARY_FIELDS + ('has_active_failures',
-                                      'has_inventory_sources'),
-    'group': DEFAULT_SUMMARY_FIELDS + ('has_active_failures',
```

Comment:
`Group` loses all of these - assuming we remove the green/red icons, I can't see any tangible reason we spend our time calculating these.

We deprecated these fields recently.

Reply:
> assuming we ... don't care about API backwards compat at /`api/v2/dashboard/`

I can't imagine that we do

> assuming we remove the green/red icons

That is a removal of user-facing functionality. There is very non-trivial meaning in those orbs, because the Ansible notion of membership includes indirect via children groups. At the same time, merely a red/green indicator is a pretty gross over-simplification. Anyway, the final call is within the UX wheelhouse.